test/lib/util/fast_json_encoder_test.exs

```diff
@@ -0,0 +1,61 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+defmodule Antikythera.FastJasonEncoderTest do
+  use Croma.TestCase
+  use ExUnitProperties
+
+  def validate(value) do
```

Comment:
```suggestion
  defp validate(value) do
```

This `validate/1` and the following `validate/2` can be private.

Reply:
Furthermore, the name `validate/1` may not be enough to make you aware that it is a test function.
What about `assert_validate/1`?

Reply:
Made private and renamed. b0c7c48381997e5ac476ebf9b332e49eab348fd4